build.gradle

```diff
@@ -82,8 +82,11 @@ repositories {
 dependencies {
 
   compile group: 'cascading', name: 'cascading-hadoop2-mr1', version: cascadingVersion, changing: true
+  compile group: 'org.apache.commons', name: 'commons-csv', version: '1.2'
+  
+  compile group: 'com.twitter.elephantbird', name: 'elephant-bird-core', version: '4.13'
```

Comment:
* common-csv is just to parse query results in HiveQueryRunner. We could get rid of it if we are happy with retrieving single-line results in String format
* elephant-bird we use it for the wrappers, we could remove this dependency by providing our own mapreduce-mapred wrappers. If we borrow some ideas from this project, how we can give them credit? Just by mentioning we've taken some code from this project? This is released under Apache 2.0